internal/web/server.go

```
@@ -287,46 +287,36 @@ func (s *Server) handleRecovery(w http.ResponseWriter, r *http.Request) {
 	// Get optional parameters
-	nonceOverrideStr := r.FormValue("nonceOverride")
-	var nonceOverride *int
-	if nonceOverrideStr != "" {
-		var nonce int
-		fmt.Sscanf(nonceOverrideStr, "%d", &nonce)
-		nonceOverride = &nonce
+	nonceOverride := 0
+	nonceOverrideProvided := false
+	if nonceOverrideStr := r.FormValue("nonceOverride"); nonceOverrideStr != "" {
+		nonceOverrideProvided = true
+		fmt.Sscanf(nonceOverrideStr, "%d", &nonceOverride)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Sscanf` is not checked

(errcheck)